RELEASE.md

```diff
@@ -1,3 +1,16 @@
+## Release 1.7.2
+### Major Features and Improvements
+> FederatedML
+* New batch strategy in coordinated Hetero LR: support masked batch data and batch shuffle
+* Model inference protection enhancement for Hetero SecureBoost with FED-EINI algorithm
+* Hetero SecureBoost supports split feature importance on host side, disables gain feature importance
+* Offline SBT Feature transform component 
+
+> Bug-Fix
+* Fixed Bug for HeteroPearson with changing default q_field value for spdz
+* Fix Data Transform's schema label name setting problem when `with_label` is False
+
+
```

Comment:
By review at the changes in the fateflow submodule below, it also changes a lot of things, such as refactoring the connection logic of the external repository, whether it is a major change needs to be explained in the release